charts/quantumleap/values.yaml

```diff
@@ -73,6 +75,7 @@ database:
     ssl: true
 
 config:
+  logLevel: "INFO"
```

Comment:
whatever you put here is serialised in the yaml file that ql load, is that what you aim at? maybe not...

Reply:
holy moly, nope that's not what I wanted to do! will fix it up, thx!!

Reply:
fixed?